packages/lesswrong/components/sequences/EAIntroCurriculum.tsx

```diff
@@ -0,0 +1,15 @@
+import { Components, registerComponent } from '../../lib/vulcan-lib';
```

Comment:
This file follows the same format as `components/sequences/CoreSequences.tsx`.

Perhaps it would be tidier for now if both pages used the same component with the `documentId` configured somewhere in the routes, but since the two pages are likely to diverge futher in the future I think it's better to keep them seperate.